docker/alpine.Dockerfile

```
@@ -11,8 +11,8 @@ RUN \
     # dev tools
     make git
 
-ARG crystal_targz
-COPY ${crystal_targz} /tmp/crystal.tar.gz
+ARG TARGETPLATFORM
+COPY crystal-${TARGETPLATFORM#linux/}.tar.gz /tmp/crystal.tar.gz
```

We should use `TARGETARCH` instead (`amd64`, `arm64`) that skips both the OS part (`linux/`) and the variant part (e.g. `/v8`).

```suggestion
COPY crystal-${TARGETARCH}.tar.gz /tmp/crystal.tar.gz
```